schemas/public/ekscluster-kfd-v1alpha2.json

```diff
@@ -1159,7 +1166,20 @@
                     "$ref": "#/$defs/Spec.Distribution.Modules.Logging.Operator"
                 }
             },
-            "required": ["opensearch"]
+            "allOf": [
+                {
+                    "if": {
+                        "properties": {
+                            "type": {
+                                "const": "opensearch"
+                            }
+                        }
+                    },
+                    "then": {
+                        "required": ["opensearch"]
+                    }
+                }
+            ]
```

Comment:
should it be a similar block but for `Loki`? If yes, remember to add it to the private schema too.